datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # CHANGELOG - datadog_checks_base
 
+## 11.0.0 / 2020-01-03
+
+* [Fixed] Ensure logs are lazily formatted. See [#5378](https://github.com/DataDog/integrations-core/pull/5378).
+* [Changed] Remove Agent 5 conditional imports. See [#5322](https://github.com/DataDog/integrations-core/pull/5322).
```

Comment:
I'm not sure we want to bump the major for this, the change should be transparent (agent 5 is on a separate branch)

Reply:
Fair enough, yes! Will revert to `Fixed` and update this PR. :+1: